### PR TITLE
about-start

### PR DESCRIPTION
**Pull request by Teslyak**
Test pull request

### CODE REVIEW

File: src/partials/about-us.html [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+      best in Swiss craftsmanship and engineering, ensuring that you'll always
+      have a reliable and stylish accessory on your wrist.
+    </p>
+    <p class="about-paragraph">
+      At our premium watch collection, we believe that a watch is more than just a
+      timekeeping device – it's a statement piece that reflects your personality
+      and style. So why settle for anything less than the best? Browse our
+      collection today and discover the perfect watch for you.
+    </p>
+    <img src="../img/Section about + buy now/watchspot.gif" alt="watchspot" />
```
Comment (OhWellNevermind) [inline src/partials/about-us.html]: wrong image path

**Review by OhWellNevermind — CHANGES_REQUESTED** [review]
fix

File: src/partials/about-us.html (hunk 1 above)
Comment (VladMCoder) [inline src/partials/about-us.html]: write right path

**Review by VladMCoder — CHANGES_REQUESTED** [review]
Write right path

**Pull request closed without merging**